### DISCUSSION TITLE
Exception thrown while executing UI block: -[RNSVGSvgView setOnClick:]: unrecognized selector sent to instance 0x10840d340

### DISCUSSION THREAD

**Discussion by faisstzera** [expo/expo#26936] [Archived: Questions]
### Minimal reproducible example

Can't reproduce in a minimal environment because it doesn't show where the error is, therefore I can't open an issue

### Summary

After upgrading to SDK 50 I've been unable to run my application, I'm getting the following error (I guess it's from the react-native-svg library) when running ONLY on the iOS simulator and it crashes immediately after opening the app:

```
Exception thrown while executing UI block: -[RNSVGSvgView setOnClick:]: unrecognized selector sent to instance 0x10840d340

RCTSurfaceStageIsPreparing
    RCTUIManager.m:1205
RCTSurfaceStageIsPreparing
FEA36690-A000-3C55-B700-9120B05AA69B
FEA36690-A000-3C55-B700-9120B05AA69B
FEA36690-A000-3C55-B700-9120B05AA69B
_dispatch_main_queue_callback_4CF
5CDC5D9A-E506-3740-B64E-BB30867B4F1B
5CDC5D9A-E506-3740-B64E-BB30867B4F1B
CFRunLoopRunSpecific
GSEventRunModal
179501B6-0FC2-344A-B969-B4E3961EBE10
UIApplicationMain
main
CB3FF411-4762-34D2-86A4-ECA13F9FB6C3
```
I can't even create a minimal reproducible example because the error does not show in which page is the problem or which parameter/svg is causing this error

### Environment

  expo-env-info 1.2.0 environment info:
    System:
      OS: Linux 4.4 Ubuntu 20.04.6 LTS (Focal Fossa)
      Shell: 5.0.17 - /bin/bash
    Binaries:
      Node: 21.1.0 - ~/.nvm/versions/node/v21.1.0/bin/node
      Yarn: 1.22.21 - ~/.nvm/versions/node/v21.1.0/bin/yarn
      npm: 10.2.4 - ~/.nvm/versions/node/v21.1.0/bin/npm
    npmPackages:
      expo: ^50.0.0 => 50.0.4
      react: 18.2.0 => 18.2.0
      react-dom: 18.2.0 => 18.2.0
      react-native: 0.73.2 => 0.73.2
      react-native-web: ~0.19.6 => 0.19.7
    npmGlobalPackages:
      eas-cli: 5.9.3
    Expo Workflow: managed
    
 Does anybody came across that error after the update? I only use one SVG in my entire project and I couldn't find anything that could cause this error

**Comment by Sourabh1095** [comment]
Getting the same error stated above. After upgrading to expo 50. Also using svg package.

**Comment by bohjak** [comment]
In case anybody is still dealing with this, in my case the issue was caused by `<TouchableWithoutFeedback>`. Replacing with `<Pressable>` fixed it.

https://github.com/software-mansion/react-native-svg/issues/2219#issuecomment-1957357059
